UN-OCHA/rwint9-site
# [RW-21] Store user change history

Refs: RW-21

This replaces https://github.com/UN-OCHA/rwint9-site/pull/439 and adds a simple custom module that store account changes in a table and use it to show the history on a user edit form for people with sufficient rights (i.e. editors).

**Note:** Also included for the future but not used in this PR, is the addition of the hooks for the reliefweb_revisions module from https://github.com/UN-OCHA/rwint9-site/pull/439 because this is a valuable improvement IMO.

### Tests

1. Checkout this branch and run `drush cim` to enable the module.
2. Log in with a user with the "User Manager" role (A)
3. Log in in another tab (for convenience) with a normal user (B)
4. As (B), edit your account (ex: change display name)
5. As (A), open (B)'s edit page, check that the history appears at the bottom with an initial record ("Account creation" message) with and another more recent record with the display name change.
6. Make so change to (B)'s account as (A) and check those appears also in the history on (B)'s edit form.

Ex:
<img width="498" alt="Screen Shot 2023-01-12 at 15 56 30" src="https://user-images.githubusercontent.com/696348/211998572-ec3368cc-9eee-49f2-ab4e-7912cec2de7a.png">
